### DISCUSSION TITLE
Componente Price para usar en slots

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1618] [Proposals]
### Goal

Disponer de un componente Mistica de importe que usar en los slots para no usar más bastardos.

### Context

Para la propuesta de Mistificación del módulo de factura del CMS necesitamos usar en el slot un importe, pero no podemos usar community y no existe en Mistica nigún slot de importe por lo que a día de hoy tenemos un bastardillo. ¿Tendría sentido incluirlo como parte de Mistica? 

<img width="531" alt="image" src="https://github.com/Telefonica/mistica-design/assets/95479574/22bea250-e73a-4493-bd5b-3ee947b0f75d">

### Impact

Añadir este componente ayudaría a Mistificar varios módulos que a día de hoy usan bastardos.

### Discussion Questions

-¿Tiene cabida en Mistica?
-¿Es mejor incluirlo como extended?

### Do you have a visual proposal?

https://www.figma.com/file/eE87dEhhkHw8esqhfvoZ0c/Figma-para-poner-ejemplos-de-cosas?type=design&node-id=241-3735&mode=design

**Comment by AnaMontes11** [comment]
Añado a la discussion esta issue en la que tratamos este tema: https://github.com/Telefonica/mistica-design/issues/960

**Comment by yceballost** [comment]
Damos por cerrada esta discussion porque ya se decidió que fuese un componente de Mística. La #960 es la tarea para esto
